doc(release): add release note for Centreon Autodiscovery 21.04.2 (#1135)
centreon/centreon-documentation: en/releases/centreon-commercial-extensions.md

## Centreon Auto Discovery

### 21.04.2

> Release date: `null`

Reviewer: ```suggestion
Release date: `September 9, 2021`
```

> Release date: `September 9, 2021`

Reviewer: ```suggestion
Release date: `November 9, 2021`
```

#### Improvements

> - Improve the way host discovery providers will be identified as compatible

Reviewer 2: ```suggestion
- Improve the way host discovery providers are identified as compatible with the current version of autodiscovery
```
```suggestion
- Improve the way host discovery providers will be identified as compatible
```
